### PR TITLE
Fix TopNav when in mobile view (size < md)

### PR DESCRIPTION
**Pull request by NduatiK**

### CODE REVIEW

File: lib/ash_admin/components/top_nav.ex [hunk 1]
```diff
@@ -103,12 +103,12 @@ defmodule AshAdmin.Components.TopNav do
       <div :if={@open} class="md:hidden" x-cloak>
         <div class="relative px-2 pt-2 pb-3 sm:px-3">
           <div class="block px-4 py-2 text-sm">
-            <.live_component
+            <ActorSelect.actor_select
```
Comment (NduatiK) [inline lib/ash_admin/components/top_nav.ex:106]: `ActorSelect` is a function component not a live component

File: lib/ash_admin/components/top_nav/drawer_dropdown.ex [hunk 2]
```diff
@@ -46,14 +46,18 @@ defmodule AshAdmin.Components.TopNav.DrawerDropdown do
             class="block px-4 py-2 text-sm hover:bg-gray-200 hover:text-gray-900"
             role="menuitem"
           >
-            {link.text}
+            <%= link.text %>
```
Comment (NduatiK) [inline lib/ash_admin/components/top_nav/drawer_dropdown.ex:49]: I think this is the surface format

File: lib/ash_admin/components/top_nav/drawer_dropdown.ex [hunk 3]
```diff
@@ -46,14 +46,18 @@ defmodule AshAdmin.Components.TopNav.DrawerDropdown do
...
         </div>
       </div>
     </div>
     """
   end
 
+  def mount(socket) do
+    {:ok, assign(socket, :open, false)}
+  end
+
```
Comment (NduatiK) [inline lib/ash_admin/components/top_nav/drawer_dropdown.ex:60]: `@open` requires a default value

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Pull request merged**